syncserver/src/web/handlers.rs

```diff
@@ -40,14 +44,35 @@ pub async fn get_collections(
     // The values below, prefixed by `_`, are temporarily and intentionally ignored at present.
     // They will be passed to the Glean logic we will implement to emit metrics.
     // We'd like for the data to be ready and in place to pass to that logic.
-    let _hashed_fxa_uid: String = meta.user_id.hashed_fxa_uid.clone();
-    let _hashed_device_id: String = meta.user_id.hashed_device_id.clone();
+    let hashed_fxa_uid: String = meta.user_id.hashed_fxa_uid.clone();
+    let hashed_device_id: String = meta.user_id.hashed_device_id.clone();
     let user_agent = request
         .headers()
         .get(header::USER_AGENT)
         .and_then(|header| header.to_str().ok())
         .unwrap_or("none");
-    let _device_info = get_device_info(user_agent);
+    let device_info: DeviceInfo = get_device_info(user_agent);
+
+    let logger: GleanEventsLogger = GleanEventsLogger {
+        // app id will be supplied when added to probe-scraper
+        app_id: "syncstorage".to_string(),
+        app_display_version: "1.0.0".to_string(),
```

Comment:
Take a look at the `logging` module on what we use for its similar values. You probably want:
```suggestion
        app_id: env!("CARGO_PKG_NAME").to_owned(),
        app_display_version: env!("CARGO_PKG_VERSION").to_owned(),
```

Reply:
`app_id` is actually what the application is defined as in the `probe-scraper` configuration: https://github.com/mozilla/probe-scraper/pull/834/files , but can use the `app_display_version` as you show above.